AlgebraInLean/Chapter01/Sheet07.lean

```diff
@@ -0,0 +1,98 @@
+import AlgebraInLean.Chapter01.Sheet06
```

Comment:
Do we really need these imports?

Reply:
Having the imports be in a single chain like this allows for subsequent chapters/sheets to blindly import the previous sheet without needing to deal with tracking down which sheets include the definitions they need or not.